src/bin/cli/wallet.js

```diff
@@ -631,50 +653,103 @@ async function initializeRequest(options = {}) {
     //         throw `Invalid RADON_BYTECODE.`
     //     }
     // }   
-    let assets
-    if (!options?.args || options.args.length === 0) {
-        assets = Object.fromEntries(
-            utils.radon.assets.search({ legacy: options?.legacy, pattern: options?.pattern, type: Witnet.RadonRequest })
-        );
-    } else {
-        assets = Object.fromEntries(
-            utils.radon.assets.search({ legacy: options?.legacy, pattern: options?.pattern, type: Witnet.RadonTemplate })
-                .filter(([,artifact]) => (
-                    artifact.homogeneous && artifact.argsCount === args.length
-                        || (args.length === 1 && template.samples[args[0]])
-                ))
-            );
+    
+    // load Radon assets from environment
+    let assets = utils.searchRadonAssets(
+        {
+            assets: loadAssets(options),
+            pattern: options?.pattern,
+        }, 
+        (key, pattern) => key.toLowerCase().indexOf(pattern.toLowerCase()) >= 0
+    );
+    
+    if (args.length > 0) {
+        // ignore RadonRequests if args were passed from the CLI
+        assets = assets.filter(([,artifact]) => !(artifact instanceof Witnet.Radon.RadonRequest));
     }
     
-    let artifact
+    // sort Radon assets alphabetically 
+    assets = assets.sort((a, b) => {
+        if (a[0] < b[0]) return -1;
+        else if (a[0] > b[0]) return 1;
+        else return 0;
+    })
```

Comment:
This is the default behavior
```suggestion
    assets = assets.sort(([a], [b]) => a - b) // or (a, b) => a[0] - b[0]
```

Reply:
Not on strings, apparently.